libs/shared/base-editor-components/src/query-output/query-output.component.spec.ts

```diff
@@ -109,6 +109,22 @@ describe('QueryOutputComponent', () => {
     expect(spy).toHaveBeenCalledWith(fullQuery);
   });
 
+  it('clicking the execute & copy button should copy and emit the correct query', () => {
+    const { page, component } = setup();
+    const copySpy = spyOn(TestBed.inject(ClipboardService), 'copyFromContent');
+    const executeSpy = spyOn(component.executeQuery, 'emit');
+
+    page.clickElement(page.executeAndCopyBtn);
+    expect(copySpy).toHaveBeenCalledWith(diffQuery);
+    expect(executeSpy).toHaveBeenCalledWith(diffQuery);
+
+    page.clickElement(page.fullQueryInput);
```

Comment:
shouldn't this additional action be checked in another separate test case? instead of 2 actions in the same test case

Reply:
I did this to be compliant with the previous two unit-tests, I mainly copy-pasted it